modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/RestSearchTemplateAction.java

```diff
@@ -68,6 +68,12 @@
             request.setScriptType(ScriptService.ScriptType.STORED);
             request.setScript(s);
         }, new ParseField("id"));
+        PARSER.declareBoolean((request, b) -> {
+            request.setExplain(b);
+        }, new ParseField("explain"));
+        PARSER.declareBoolean((request, b) -> {
+            request.setProfile(b);
```

Comment:
can this be `PARSER.declareBoolean(SearchTemplateRequest:: setProfile);` instead